server/server.go

```diff
@@ -123,7 +120,7 @@ func NewServer(ctx context.Context, prof Profile, logger *zap.Logger, loggerLeve
 	}
 	s.db = storeDB
 	defer func() {
-		if !s.boot {
+		if !start {
 			_ = s.metaDB.Close()
```

Comment:
Second thought: `metaDB` and `storeDB` are all closed in the `Shutdown()` method.
Why bother closing it in this function?

Reply:
SG. We can use defer likes below?
```go
defer func(){
    if !serverStarted{
        _ = s.Shutdown()
    }
}